composer.json

```diff
@@ -81,6 +81,7 @@
         "html2text/html2text": "^4.3.1",
         "incenteev/composer-parameter-handler": "^2.2",
         "j0k3r/graby": "^2.4.5",
+        "j0k3r/graby-site-config": "^1.0",
```

Comment:
just curious, why not dev-master?

Reply:
I don't know, just to follow the monthly release instead of having a dependabot PR every time the master change